### PR TITLE
Feat/move base flex

### PR DESCRIPTION
**Pull request by dnikitop**

### CODE REVIEW

**Review by grieneis — COMMENTED** [review]
Seems good

File: move_backwards_recovery/src/move_backwards_recovery.cpp [hunk 1]
```diff
@@ -115,24 +115,6 @@ void MoveBackRecovery::runBehavior(){
   }
   runOnce_ = true;
```
Comment (ccpowers) [inline move_backwards_recovery/src/move_backwards_recovery.cpp:117]: need to compare this to the implementation in mfp_executive to make sure the logic is the same

Comment (dnikitop) [reply move_backwards_recovery/src/move_backwards_recovery.cpp:117]: I went through, and it seems like it does

**Review by mbittarelli6river — DISMISSED** [review]

File: .vscode/c_cpp_properties.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (ccpowers) [inline .vscode/c_cpp_properties.json]: gotta .gitignore these

File: base_local_planner/src/odometry_helper_ros.cpp [hunk 3]
```diff
@@ -204,15 +204,15 @@ nav_msgs::Odometry OdometryHelperRos::forwardEstimateOdometry(geometry_msgs::Twi
   double t = 0;
   while (t < estimate_dt)
   {
-    ROS_DEBUG("updating from [%f, %f, %f] and vel from [%f, %f]",
-      pos[0], pos[1], pos[2], vel[0], vel[2]);
+    //ROS_DEBUG("updating from [%f, %f, %f] and vel from [%f, %f]",
```
Comment (ccpowers) [inline base_local_planner/src/odometry_helper_ros.cpp]: commented out code

**Review by ccpowers — APPROVED** [review]

**Pull request merged**